src/components/DownloadMailingListButton.tsx

```diff
@@ -1,25 +1,40 @@
 import React from 'react'
-import { UnprocessedResultsFromCRM } from '../types'
+import { UnprocessedResultsFromCRM, OwnerType } from '../types'
 
 type DownloadButtonProps = {
     results: UnprocessedResultsFromCRM[]
 }
+const key = 'id'
+
 export function DownloadMailingListButton (props: DownloadButtonProps) {
     let downloadUrl = null
     let csvData = '"Contact Name","Contact Type","Mailing Street Address","Mailing Suburb","Mailing State","Mailing Postcode","Property Address","Property Type (Marketing)","Company"\r\n'
 
-    props.results.forEach((result) => {
-        if (typeof result.owner_details !== 'undefined' && Array.isArray(result.owner_details)) {
-            result.owner_details.forEach((owner) => {
-                const propertyAddress = result.Deal_Name
-                const company = owner.Company || '-'
-                const ownerAddress = owner.Postal_Address || 'Address not available'
-                const newRow = `"${owner.Name}","${owner.Contact_Type}","${ownerAddress}","${owner.Postal_Suburb}","${owner.Postal_State}","${owner.Postal_Postcode}","${propertyAddress}","${result.Property_Category_Mailing.join(';')}","${company}"\r\n`
-                csvData += newRow.replace(/null/g, '-')
-            })
-        }
-    })
+    const objectArr = props.results
```

Comment:
What is a more descriptive name for this variable? objectArr is too generic and doesn't give any context.